cristal-integration-ui/pom.xml

```
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.xwiki.contrib.cristal.integration</groupId>
+    <artifactId>cristal-integration</artifactId>
+    <version>1.1.2-SNAPSHOT</version>
+  </parent>
+  <artifactId>cristal-integration-ui</artifactId>
+  <packaging>xar</packaging>
+  <name>Cristal Integration - UI</name>
+  <description>UI elements to interact with the embedded Cristal instance</description>
+
+  <properties>
+    <xwiki.extension.category>application</xwiki.extension.category>
+    <xwiki.extension.name>Cristal Integration - UI</xwiki.extension.name>
```

The `xwiki.extension.name` must be user friendly.


```suggestion
    <xwiki.extension.name>Cristal Integration Application</xwiki.extension.name>
```